linkml/utils/mergeutils.py

```diff
@@ -1,4 +1,5 @@
 import dataclasses
+import logging
```

Comment:
Probably a good idea to follow syntax like this: https://github.com/linkml/linkml/blob/main/linkml/generators/linkmlgen.py#L14?